### PR TITLE
Share base template for opportunity and campaign pages

### PR DESCRIPTION
**Pull request by gvn**
This achieves a few things:

- Opportunity and Campaign templates now share a base (More DRY)
- The "guts" of the Opportunity template are no longer generated from Pug. You can just edit `landingpage.html` directly now that it's no longer compiled.
- All Pug and Django pages will share common header/footer code (`source/pug/templates/master.pug`)

### CODE REVIEW

File: source/pug/templates/base-for-django.pug [hunk 1]
```diff
@@ -0,0 +1,18 @@
+//-
+  This template is used to generate HTML for usage in Mezzanine.
+  There are {{ TOKENS }} in place for Mezzanine to target with CMS data.
+  `npm build` will compile this to `/dest/opportunity/template/index.html`
```
Comment (mmmavis) [inline source/pug/templates/base-for-django.pug]: is this line true for campaign pages too?

Comment (gvn) [reply source/pug/templates/base-for-django.pug]: Ah I need to update this...not quite accurate

File: source/pug/templates/base-for-django.pug [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+extends ../templates/master.pug
+
+block masterParams
+  - let isMezzanine = true
+  - let pageTitle = `{{ page.meta_title }}`;
+
+include ../partials/sign-up-box.pug
```
Comment (alanmoo) [inline source/pug/templates/base-for-django.pug]: What's this doing?

Comment (gvn) [reply source/pug/templates/base-for-django.pug]: Ah, I think actually nothing. That was an artifact from the previous opportunity template, but it's an unused mixin apparently.

**Pull request merged**